jaxrs-api/src/main/java/jakarta/ws/rs/client/ClientRequestContext.java

```diff
@@ -88,7 +88,7 @@
      * the given name.
      * @see #getPropertyNames()
      */
-    default public boolean hasProperty(String name) {
+    public default boolean hasProperty(String name) {
```

Comment:
I personally prefer having `default` being the first word as still people often are not used with default methods, so they clearly see it. Is it enforced by Jakarta EE to have it this way, then I'm fine with it. Otherwise I would plea not to change it.

Reply:
You might want to ask what happens if you remove the "public" modifier entirely.

If nothing happens, why is the modifier there to begin with?

Reply:
@mkarg That ordering is defined by the `ModifierOrder` rule in checkstyle. Using checkstyle rules is about compromising on personal preference for the sake of consistency.

Reply:
@arjantijms There's many places in the API where `public` is redundant (this whole file being of those). I think it would be better to fix that on a separate PR.

Reply:
Yes, would be good in a separate PR indeed, thx!